dbot.py

```diff
@@ -361,6 +364,36 @@ async def on_command(ctx):
             "cmd: chan {0.message.channel} ({0.message.channel.id}), serv {0.message.guild} ({0.message.guild.id}), "
             "auth {0.message.author} ({0.message.author.id}): {0.message.content}".format(ctx)
         )
+        print(
+            "cmd: chan {0.message.channel} ({0.message.channel.id}), serv {0.message.guild} ({0.message.guild.id}), "
+            "auth {0.message.author} ({0.message.author.id}): {0.message.content}".format(ctx)
+        )
+        # send command to kafka
+        producer = KafkaProducer(KafkaProducer.config)
+        avrae_command = {
+            "EVENT_TIME": time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime()),
+            "PLATFORM": "discord",
+            "MESSAGE_ID": ctx.message.id,
+            "MESSAGE_NAME": "AVRAE_COMMAND",
+            "DISCORD_ID": ctx.message.author.id,
+            "DDB_USER_ID": ctx.message.author.name,
+            "DISCORD_SERVER_ID": ctx.message.guild.id,
```

Comment:
not sure if this is intended, but this will not produce an event in case of a command in DM.

Reply:
@kevklash , Yeah, as Turtle states, you will want to take the ctx.message.guild.id and give it a default value, or accept None in Kafka's side.